project-base/src/Shopsys/ShopBundle/Resources/config/services/data_fixtures.yml

```diff
@@ -1,7 +1,45 @@
 services:
     _defaults:
-        tags: ['doctrine.fixture.orm']
         autowire: true
 
     Shopsys\ShopBundle\DataFixtures\:
-        resource: '../../../DataFixtures/**/*DataFixture.php'
+        tags: ['doctrine.fixture.orm']
+        resource: '../../../DataFixtures/**/*{DataFixture,Loader}.php'
+
+    Shopsys\ShopBundle\DataFixtures\ProductDataFixtureReferenceInjector: ~
+
+    Shopsys\ShopBundle\DataFixtures\Demo\ProductDataFixtureLoader: ~
```

Comment:
If I understand correctly `Shopsys\ShopBundle\DataFixtures\Demo\ProductDataFixtureLoader` and `Shopsys\ShopBundle\DataFixtures\Demo\ProductParametersFixtureLoader` shouldn't be here as they are covered by `resource: '../../../DataFixtures/**/*{DataFixture,Loader}.php'`. Am I right?

Reply:
yes, that is right.